util/dvsim/FlowCfg.py

```diff
@@ -473,6 +474,18 @@ def _get_results_page_link(self, link_text):
             self.results_server_prefix, self.results_server_url_prefix)
         return "[%s](%s)" % (link_text, results_page_url)
 
+    def gen_email_html_summary(self):
+        if self.is_master_cfg:
+            gen_results = self.results_summary_md
```

Comment:
Wondering if this should also include individual cfg summary.

Reply:
I remembered our discussion about it. The only concern to add the individual cfg is: if there are lots of failures, it will be a very long email.

If the emailed result is also published online (which in most case should be - for nightly regression), then recipients should be able to click one the links to see individual cfg result.

But personally I do not mind either way :)

Reply:
Aah, yes, you're right. the summary links to the individual reports. Do you want to testdrive this and see what the email looks like? 
We may want to inline the contents of `util/dvsim/style.css` in the email html.

Reply:
Sounds good, will give it a try :)